src/main/osd/osd.c

```diff
@@ -90,6 +90,9 @@
 #include "hardware_revision.h"
 #endif
 
+#define VIDEO_BUFFER_CHARS_PAL    480
```

Comment:
This is already defined in `drivers/max7456.h`. It would be better to include that header rather then redefining it again in a different place.